setup.py

```diff
@@ -7,7 +7,7 @@
     long_description=('Please visit https://github.com/unt-libraries/django-edtf'
                       ' for the latest documentation.'),
     install_requires=[
-        'edtf-validate @ git+https://github.com/unt-libraries/edtf-validate.git@master'],
+        'edtf-validate @ git+https://github.com/unt-libraries/edtf-validate.git@py3'],
```

Comment:
should leave this with master and wait to merge until the edtf-validate pr that is open is merged.

Reply:
ok!